sys/build.rs

```diff
@@ -1,31 +1,239 @@
-extern crate pkg_config;
+use semver::Version;
```

Comment:
Just one thing you need to handle: to not link when using the `dox` feature (since we're generating documentation, we don't care if the library is actually there with the right version or not).

Reply:
What I meant is that if we are using rustdoc (`cfg(doc)`) and not rustc, we should not link the libraries.